cfme/cloud/instance/openstack.py

```diff
@@ -124,15 +124,22 @@ class form(View):  # noqa
 
 
 class DetachVolumeView(CloudInstanceView):
-    @View.nested
-    class form(View):  # noqa
-        volume = BootstrapSelect('volume_id')
-        submit_button = Button('Submit')
-        cancel_button = Button('Cancel')
+    volume = BootstrapSelect('volume_id')
+    submit_button = Button('Submit')
+    cancel_button = Button('Cancel')
 
     # Only the instance name is displayed, cannot confirm provider
     is_displayed = displayed_not_implemented
 
+    def after_fill(self, was_change):
+        # TODO: BZ1647695
+        if was_change:
```

Comment:
Moving the `self.submit_button.click()` out of this, the `if` statement should be updated to:
```if was_change and BZ(BZ1647695, forced_streams=['5.10', 'upstream']).blocks```

That way when the bug is fixed, we no longer have to worry about this workaround.